File: Service/Implement/CompanyExaminationQuestionsService.cs
namespace Service.Implement
{
    public class CompanyExaminationQuestionsService : BaseService<CompanyExaminationQuestions, ICompanyExaminationQuestionsRepository>
    , ICompanyExaminationQuestionsService
    {
    private readonly ICompanyExaminationQuestionsRepository _CompanyExaminationQuestionsRepository;
    public CompanyExaminationQuestionsService(ICompanyExaminationQuestionsRepository CompanyExaminationQuestionsRepository) : base(CompanyExaminationQuestionsRepository)
    {
    _CompanyExaminationQuestionsRepository = CompanyExaminationQuestionsRepository;
    }
    }
    }

File: CMS/src/app/shared/CompanyExaminationQuestions.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CompanyExaminationQuestions } from 'src/app/shared/CompanyExaminationQuestions.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CompanyExaminationQuestionsService extends BaseService{

    List: CompanyExaminationQuestions[] | undefined;
    ListFilter: CompanyExaminationQuestions[] | undefined;
    FormData!: CompanyExaminationQuestions;

    DisplayColumns001: string[] = ['STT', 'CauHoiATTPID', 'Save'];
    DisplayColumns002: string[] = ['STT', 'Name'];
    DisplayColumns003: string[] = ['Name'];

    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CompanyExaminationQuestions";
    }
}

